### ISSUE TITLE
Scan code stopped 

### ISSUE THREAD

**Issue by nicearma** [pdliuw/ai_barcode#7]
This widget is nice, but why the scan code is stopped with the first result? i think the scan have to continue and leave the choice to the user to keep going if he want

**Comment by pdliuw** [comment]
We will change the stop scanning from the program automatically stop scanning to the user to control whether to stop scanning, so stay tuned

**Comment by alexaf2000** [comment]
There is some new? Is a really wanted functionality

**Comment by kuntash34** [comment]
@pdliuw @nicearma @alexaf2000  Yes I have the same issue, I want to scan list of barcode and store in list but I don't want to close camera at the same time. is there any workaround or solution to this.

**Comment by CodingWithTashi** [comment]
Hi, I have tested out the code for android part and it seems this package uses https://github.com/dm77/barcodescanner and if you want to continue scanning barcode you will have to call `mScannerView.resumeCameraPreview(this);` in kotlin but in flutter it can by done by calling.   
```
scannerController = ScannerController(scannerResult: (result) {
     `_scannerController.startCameraPreview();`
    } 
```    
`_scannerController.startCameraPreview();` internally call `mScannerView.resumeCameraPreview(this);`
